client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -63,13 +83,31 @@ function ItemDetail({
     }
     // Everytime we change to a 'new' item, refresh removal state flag
     setIsRemoving(false);
+    // Refresh whether we collapse media (image) to show more of text
+    setMediaCollapsed(false);
+    // Determine whether we should render 'show more' expand button
+    const contentDiv = document.getElementById("item-detail-description");
+    if (contentDiv) setIsOverflow(checkContentOverflow(contentDiv));
+    else setIsOverflow(false); // Cannot find content tag, no content to render
+    // Compute the margin required to offset minHeight for footer
+    const footerMarginOffset =
+      footerMinHeight -
+      document.documentElement.clientHeight * footerPercentageHeight;
+    if (footerMarginOffset > 0)
+      setMinHeightMarginTop(
+        Math.floor(
+          (100 * footerMarginOffset) / document.documentElement.clientHeight
+        )
+      );
+    else setMinHeightMarginTop(0);
```

Comment:
same for this. I did not find "minHeightMarginTop" being used anywhere, is it needed?

Reply:
ah, obsolete, I forgot to remove this.
Now removed.